Sources/SQLiteKit/SQLiteRow+SQLRow.swift

```diff
@@ -14,10 +14,10 @@ extension SQLiteRow: SQLRow {
     }
 
     public func contains(column: String) -> Bool {
-        return self.column(column) != nil
+        self.column(column) != nil
     }
 
-    public func decode<D>(column: String, as type: D.Type) throws -> D where D : Decodable {
+    public func decode<D: Decodable>(column: String, as: D.Type) throws -> D {
```

Comment:
You could also do stuff like `(some Decodable).Type` but to be honest not sure if the current approach isn't better/more-readable.

Reply:
Nope. The type has to be referred to by its generic name (`D`) at least twice in the signature (the parameter and the return type), as well as an additional time in the method body; `some` is only useful where you never need to refer directly to the type by generic name. The usage in the method body could be gotten around by referring to the parameter by its label, but there's no avoiding that you have to express it more than once in the signature. (Oh, and writing `some Decodable` twice would be the equivalent of `<T: Decodable, U: Decodable>`, which is not what's needed here.)

Reply:
ah right didn't notice the `-> D`